src/main/java/carina/learning/api/products/GetTokenInBearerMethod.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright 2020-2023 Zebrunner Inc (https://www.zebrunner.com).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package carina.learning.api.products;
+
+import com.zebrunner.carina.api.AbstractApiMethodV2;
+import com.zebrunner.carina.api.annotation.Endpoint;
+import com.zebrunner.carina.api.annotation.SuccessfulHttpStatus;
+import com.zebrunner.carina.api.http.HttpMethodType;
+import com.zebrunner.carina.api.http.HttpResponseStatusType;
+import com.zebrunner.carina.utils.config.Configuration;
+import org.apache.http.HttpHeaders;
+
+@Endpoint(url = "${base_url}/auth/me", methodType = HttpMethodType.GET)
+@SuccessfulHttpStatus(status = HttpResponseStatusType.OK_200)
+public class GetTokenInBearerMethod extends AbstractApiMethodV2 {
```

Comment:
```suggestion
public class GetCurrentAuthUserDataMethod extends AbstractApiMethodV2 {
```
<img width="479" alt="image" src="https://github.com/NadiMarkava/carina-learning/assets/103670717/bda5f218-7c78-4159-89fd-335c75316c8b">

Reply:
changed name of class